tests/adapters_t/sites_t/test_moab.py

```diff
@@ -80,6 +80,51 @@
 
 """
 
+TEST_RESOURCE_STATE_TRANSLATION_RESPONSE = f"\n\n".join(
+    f"""
+<Data>
+ <Object>queue</Object>
+ <cluster LocalActiveNodes="0" LocalAllocProcs="0" LocalConfigNodes="918" LocalIdleNodes="9" LocalIdleProcs="1336" LocalUpNodes="916" LocalUpProcs="18776" RemoteActiveNodes="0" RemoteAllocProcs="0" RemoteConfigNodes="0" RemoteIdleNodes="0" RemoteIdleProcs="0" RemoteUpNodes="0" RemoteUpProcs="0" time="1583334681"/>
+ <queue count="0" option="active"/>
+ <queue count="0" option="eligible"/>
+ <queue count="0" option="blocked"/>
+</Data>
+
+<Data>
+ <Object>queue</Object>
+ <cluster LocalActiveNodes="0" LocalAllocProcs="0" LocalConfigNodes="918" LocalIdleNodes="9" LocalIdleProcs="1336" LocalUpNodes="916" LocalUpProcs="18776" RemoteActiveNodes="0" RemoteAllocProcs="0" RemoteConfigNodes="0" RemoteIdleNodes="0" RemoteIdleProcs="0" RemoteUpNodes="0" RemoteUpProcs="0" time="1583334681"/>
+ <queue count="1" option="completed" purgetime="86400">
+  <job Account="bw16g013" CompletionCode="CNCLD" EEDuration="2729" GJID="76242{num:02}" Group="ka_etp" JobID="76242{num:02}" JobName="startVM.py" ReqAWDuration="360" ReqProcs="20" StartTime="0" StatPSDed="0.000000" StatPSUtl="0.000000" State="{resource_status}" SubmissionTime="1583331813" SuspendDuration="0" User="ka_qb1555"/>
+ </queue>
+</Data>
+"""
+    for num, resource_status in enumerate(
+        [
```

Comment:
Please factor out state/status definitions, and re-use the same definition both for the mock response and the test. I'd recommend a list of tuples, e.g. ``[("BatchHold", ResourceStatus.Stopped), ("Canceling", ResourceStatus.Running), ...]`` (all our targets have order preserving dicts, but it's the more appropriate data structure).

Reply:
Agreed!

Reply:
Done 👍